packages/dashboard/src/pages/Wallets/hooks/useWalletContext.ts

```diff
@@ -0,0 +1,7 @@
+import { createContainer } from 'unstated-next'
+
+function useWalletContext() {
+    return {}
+}
+
+export const WalletContext = createContainer(useWalletContext)
```

Comment:
What is this file used for ?

Reply:
> What is this file used for ?

Oh, this file used for wallet page in dashboard. Please ignore it, it will be used later